### PR TITLE
3.4.0

### PR DESCRIPTION
**Pull request by Ashuaidehao**

### CODE REVIEW

**Review by erikzhang — DISMISSED** [review]

**Review by Liaojinghui — DISMISSED** [review]

**Comment by Liaojinghui** [comment]
No sure whether this is an issue or not, but there is a version conflict.

  OracleService.csproj: [NU1605] Detected package downgrade: Neo from 3.3.1 to 3.3.1-CI01377. Reference the package directly from the project to select a different version. 
 Neo.Plugins.OracleService -> NeoFS.API 3.3.1 -> Neo (>= 3.3.1) 
 Neo.Plugins.OracleService -> Neo (>= 3.3.1-CI01377)

**Comment by erikzhang** [comment]
`NeoFS.API` should not be updated if there is no change in it.

**Comment by superboyiii** [comment]
> No sure whether this is an issue or not, but there is a version conflict.
> 
> OracleService.csproj: [NU1605] Detected package downgrade: Neo from 3.3.1 to 3.3.1-CI01377. Reference the package directly from the project to select a different version. Neo.Plugins.OracleService -> NeoFS.API 3.3.1 -> Neo (>= 3.3.1) Neo.Plugins.OracleService -> Neo (>= 3.3.1-CI01377)

Will update NeoFS.API after Neo v3.4.0 released.

**Review by shargon — DISMISSED** [review]

**Review by superboyiii — DISMISSED** [review]

File: src/RpcClient/Utility.cs [hunk 1]
```diff
@@ -18,6 +18,7 @@
 using System;
 using System.Linq;
 using System.Numerics;
+using Neo.Json;
```
Comment (superboyiii) [inline src/RpcClient/Utility.cs:21]:
```suggestion
```

File: src/OracleService/OracleService.csproj [hunk 2]
```diff
@@ -7,6 +7,7 @@
   <ItemGroup>
     <PackageReference Include="Neo.ConsoleService" Version="1.2.0" />
     <PackageReference Include="NeoFS.API" Version="3.3.1" />
+    <PackageReference Include="Neo" Version="3.3.1" />
```
Comment (superboyiii) [inline src/OracleService/OracleService.csproj:10]:
```suggestion
    <PackageReference Include="Neo" Version="3.3.1-CI01380" />
```

**Comment by Liaojinghui** [comment]
Need a new version of plugins to test neo-node. Please merge this pr.

**Comment by superboyiii** [comment]
> Need a new version of plugins to test neo-node. Please merge this pr.

Let's wait for neo v3.4.0

**Comment by erikzhang** [comment]
Need to fix NeoFS.API.

**Review by superboyiii — APPROVED** [review]

**Review by vncoelho — APPROVED** [review]

**Review by erikzhang — APPROVED** [review]

**Pull request merged**